src/app/(fullscreen)/event/success/_components/client.tsx

```
@@ -0,0 +1,27 @@
+  const router = useRouter();
+
+  return (
+    <FixedLayout
+      buttonText={step === 1 ? "이벤트 미리보기" : "이벤트 게시하기"}
+      showCloseButton={true}
+      onButtonClick={() => {
+        setStep(step + 1);
+      }}
+      detail={step === 2 ? true : false}
+    >
+      {step === 1 && <Step1 />}
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)